test/fetch-mock-helper.js

```diff
@@ -0,0 +1,8 @@
+export default function fixtureWithHeaders(fixture) {
```

Comment:
Do we need a test to test this helper function?

Reply:
That doesn't seem like a bad idea. Technically, if this breaks, it could invalidate a whole whack of tests. I think tho, with the more recent change to the client (ignoring headers and trying to parse json and falling back to text), this method will probably become unnecessary. @rebeccajfriedman, is that correct?

Reply:
@minasmart you are right. We are going to hold off on merging that PR in `graphql-js-client` because we aren't so happy with the resultant code, and we were struggling to come up with a better approach.